plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/SonarQubeReader.java

```diff
@@ -210,347 +210,277 @@ public static int cweLookup(String squidNumber) {
-            case "S2386":
-                return 607; // Mutable fields should not be "public static"
-            case "S2441":
-                return 579; // Non-serializable objects should not be stored in"HttpSessions"
-            case "S2479":
-                return CweNumber
-                        .DONTCARE; // Whitespace and control characters in literals should be
-                // explicit
-            case "S2583":
-                return 489; // Conditions should not unconditionally evaluate to"TRUE" or to"FALSE"
-            case "S2589":
-                return CweNumber
-                        .DONTCARE; // Boolean expressions should not be gratuitous - CWEs: 570/571
-            case "S2658":
-                return 470; // Use of Externally-Controlled Input to Select Classes or Code ('Unsafe
+                return CweNumber.RSA_MISSING_PADDING; // Cryptographic RSA algorithms should always
+                // incorporate OAEP (Optimal Asymmetric Encryption
+                // Padding)
+            case "S2278": // Benchmark Vuln: DES (Data Encryption Standard) and DESede (3DES) should
+                // not be used
+                return CweNumber.WEAK_CRYPTO_ALGO;
+            case "S2293": // The diamond operator ("<>") should be used
+                return CweNumber.DONTCARE;
+            case "S2384": // Mutable members should not be stored or returned directly
+                return CweNumber.PASS_MUTABLE_OBJECT_TO_UNTRUSTED_MODULE;
+            case "S2386": // Mutable fields should not be "public static"
+                return CweNumber.PUBLIC_STATIC_FINAL_MUTABLE_OBJECT;
+            case "S2441": // Non-serializable objects should not be stored in"HttpSessions"
+                return CweNumber.NON_SERIALIZABLE_OBJECT_IN_SESSION;
+            case "S2479": // Whitespace and control characters in literals should be explicit
+                return CweNumber.DONTCARE;
+            case "S2583": // Conditions should not unconditionally evaluate to"TRUE" or to"FALSE"
+                return CweNumber.ACTIVE_DEBUG_CODE;
+            case "S2589": // Boolean expressions should not be gratuitous - CWEs: 570/571
+                return CweNumber.DONTCARE;
+            case "S2658": // Use of Externally-Controlled Input to Select Classes or Code ('Unsafe
                 // Reflection')
-            case "S2677":
-                return CweNumber.DONTCARE; // "read" and "readLine" return values should be used
-            case "S2681":
-                return 483; // Multiline blocks should be enclosed in curly braces
+                return CweNumber.UNSAFE_REFLECTION;
+            case "S2677": // "read" and "readLine" return values should be used
+                return CweNumber.DONTCARE;
+            case "S2681": // Multiline blocks should be enclosed in curly braces
+                return CweNumber.INCORRECT_BLOCK_DELIMITATION;
             case "S2696":
```

Comment:
For these 3, can you move the comments up to match similar changes throughout?

Reply:
Done